src/ts/packages.ts

```diff
@@ -1,4 +1,5 @@
 $(".section-registry").on("filterSelect", (event) => {
```

Comment:
This jQuery event-handling stuff was just me trying to be pragmatic; time-permitting I'd like to pull it up into a wrapper component, now that things are beginning to settle.